healthcheck/install.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const routes = require('./routes')
+const outputs = require('./outputs')
+
+function addTo(app, config) {
+  console.log("adding healthCheck ")
+  app.get('/health', routes.configure(config))
+  console.log("adding liveness")
+  app.get('/health/liveness', (req, res) => res.status(200).json(outputs.status(outputs.UP)))
```

Comment:
is there a test for this? (I couldn't see one)

Reply:
just added the file installTest.js